streams/src/main/java/org/apache/kafka/streams/processor/internals/Task.java

```diff
@@ -44,8 +44,7 @@
 
     void resume();
 
-    void closeSuspended(final boolean clean,
-                        final boolean isZombie,
+    void closeSuspended(boolean clean, final boolean isZombie,
```

Comment:
Why this reformatting? Did you intent to remove `clean` ?

Reply:
`clean` in `closeSuspended` is only used to determine if we can write a checkpoint file. Since we are not writing checkpoint files any more I think we can remove it.